cnxarchive/sql/get-module-metadata.sql

```diff
@@ -19,7 +19,7 @@ FROM (SELECT
   iso8601(m.created) AS created, iso8601(m.revised) AS revised,
   m.stateid, m.doctype,
   (SELECT row_to_json(license) AS license FROM (
-        SELECT l.code, l.version, l.name, l.url
+        SELECT l.code AS abbr, l.version, l.name, l.url
```

Comment:
ISTM the term of art for CC license short codes is that: short code, or code, for short. Oh well.